bigtop-tests/test-artifacts/hadoop/src/main/groovy/org/apache/bigtop/itest/hadoop/mapreduce/TestHadoopExamples.groovy

```diff
@@ -152,9 +143,7 @@ class TestHadoopExamples {
   public static LinkedHashMap<String, Object[]> generateTests() {
     LinkedHashMap<String, Object[]> res = [:];
     examples.each { k, v -> res[k] = [k.toString(), v.toString()] as Object[]; }
-    if (HADOOP_COMMAND != "hadoop-qfs") {
-      additional_examples.each { k, v -> res[k] = [k.toString(), v.toString()] as Object[]; }
-    }
+    additional_examples.each { k, v -> res[k] = [k.toString(), v.toString()] as Object[]; }
```

Comment:
We can merge `additional_examples` into `examples`, because the former was separated from the latter so as to addressing QFS test failure originally. See the comment at line 129.

Reply:
Sure. I merged the maps.